js/form/src/Dropdown.svelte

```diff
@@ -86,6 +87,33 @@
 		e.preventDefault();
 	}
 
+	function handle_blur(e: FocusEvent) {
+		if (multiselect) {
+			inputValue = "";
+		} else if (!allow_custom_value) {
+			if (value !== inputValue) {
+				if (typeof value === "string" && inputValue == "") {
+					inputValue = value;
+				} else {
+					value = undefined;
+					inputValue = "";
+				}
+			}
+		}
+		showOptions = false;
+		dispatch("blur");
+	}
+
+	function handle_focus(e: FocusEvent){
+		dispatch("focus");
+		showOptions = !showOptions;
+		if (showOptions) {
```

Comment:
there's an existing handleFocus method, that is pretty identical except you changed
`filtered = choices;`
to
`inputValue = "";`

any reason for this?

Reply:
We should keep `filtered = choices;` I believe, otherwise, it'll revert some of the changes in https://github.com/gradio-app/gradio/pull/5039